Style for the modal contents

carlosms opened the pull request:
Part of #48.

There is a small difference with the design spec: the blue background of the code line gutter does not carry all the way to the top.
The reason is that the width is not fixed, it depends on the number of lines, it has different widths for 1 or 3 digit line numbers. There must be a convoluted way to make it work, but I think it's not worth it, at least for now.

![a](https://user-images.githubusercontent.com/1469173/41969939-5a5e9d50-7a09-11e8-9745-fd70bc5d8b37.png)

![b](https://user-images.githubusercontent.com/1469173/41969943-5ce6a3ba-7a09-11e8-8c32-2b6e104cf2f8.png)

![c](https://user-images.githubusercontent.com/1469173/41969947-5ee8e54c-7a09-11e8-80f4-84422f18c3ce.png)

smacker commented on frontend/src/App.less:
```diff
@@ -85,12 +107,6 @@ body {
 .modal-body {
     height: ~'calc(100vh - 150px)';
 
-    .uast-editor,
```
are you sure we don't need it?

I remember one of the reasons to add it was "This site appears to use a scroll-linked positioning effect. This may not work well with asynchronous panning; see https://developer.mozilla.org/docs/Mozilla/Performance/ScrollLinkedEffects for further details and to join the discussion on related tools and features!".

smacker replied:
nwm. I see you moved it to another file.

smacker commented on frontend/src/variables.less:
```diff
@@ -234,3 +254,10 @@
+    .icon-btn-variant(@primary);
+
+    float: right;
+}
```
new line

carlosms commented:
Thanks for the comments. I've checked and firefox does not complain in the console about the scroll, so it looks things work as they should.

Rebased and squashed the fixe for the new line. Let's merge :)
